src/components/Action/Action.vue

```diff
@@ -77,7 +77,7 @@ export default {
      * Button type
      * @values type attr of the button tag
      */
-    buttonType: { type: String, default: 'submit' },
+    buttonType: { type: String, default: 'submit' }
```

Comment:
Did you change prettier settings? It's removing trailing commas 🙄

Reply:
I was wondering about this myself actually. I didn't do anything related to prettier... 🤔
In my vscode settings I have this:
```json
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
```

And I also turned on format-on-save. For Swimm webapp it works fine 🤷‍♂️ 

According to our configuration, should we have trailing commas? How can I determine whether or not this is a problem?